scripts/migrate_public_files_nodes.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# encoding: utf-8
+"""Migrate Share Windows to existing users
+
+Run dry run: python -m scripts.migrate_public_files_nodes --dry
+Run migration: python -m scripts.migrate_public_files_nodes
+
+"""
+import sys
+import logging
+
+from website.app import init_app
+from framework.auth import core
+from website.public_files import give_user_public_files_node
+from framework.transactions.context import TokuTransaction
+
+from scripts import utils as script_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def main(dry_run):
+    users = core.User.find()
+
+    for user in users:
+        if user.is_registered and user.public_files_node is None:
```

Comment:
Will a public files node be created for existing unregistered users when they register?